docs/bundle/multiple_queues.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+parent: "Symfony bundle"
+title: Functional testing
```

Comment:
You probably left the title here by accident.

Reply:
Indeed. Will be fixed asap